README.md

```diff
@@ -73,3 +76,12 @@ If you do that, the code is running directly on your machine, without any isolat
 ## Troubleshooting
 
 If you experience an issue or want to propose a new feature in WBO, please [open a github issue](https://github.com/lovasoa/whitebophir/issues/new).
+
+
+### Additional Features
+#### Two new tools:
+- Background (Shortcut: `b`): Allows you to upload an image for the background of the canvas. 
+- Pointer (Shortcut: `f`): Allows users to point at things for others to see.
+
+#### Minio suport:
+- Instead of writing to the filesystem, the server now saves boards to a minio instance. The configuration is inside  [server/minio.js](server/minio.js).
```

Comment:
I don't think we want the changes to the README

Reply:
Yup, it's in my TODO list to just remove these.